pkg/internal/codegen/parse/util.go

```diff
@@ -40,6 +40,8 @@ const (
 	printColumnFormat  = "format"
 	printColumnPri     = "priority"
 	printColumnError   = "invalid printcolumn path. name,type, and JSONPath are required kye-value pairs and rest of the fields are optinal. For example: // +kubebuilder:printcolumn:name=abc,type=string,JSONPath=status"
+	shortnamePrefix    = "+kubebuilder:shortname:"
+	shortNameError     = "invalid shortname params. Shortname should be a no-empty string and is seperated by comma. For example: // " + shortnamePrefix + "deploy,dpl"
```

Comment:
nit: `non-empty string`

Reply:
also could just write `non-empty comma-separated list of names`

Reply:
@DirectXMan12 thanks for your time to review this pr!  the sentence was modified as your request